src/solvers/dg_curved/dg_1d.jl

```diff
@@ -54,20 +61,20 @@ end
 
 # TODO: Taal dimension agnostic
 function calc_boundary_flux!(cache, u, t, boundary_condition::BoundaryConditionPeriodic,
-                             equations::AbstractEquations{1}, mesh::CurvedMesh{1}, dg::DG)
+                             mesh::CurvedMesh{1}, equations::AbstractEquations{1}, dg::DG)
```

Comment:
No big deal, but we should probably dispatch on just one of the arguments to get the dimension right (preferably the one we use elsewhere)

Reply:
Yes, definitely. See https://github.com/trixi-framework/Trixi.jl/pull/550#discussion_r615286816